plugins/source/gcp/docs/tables/gcp_websecurityscanner_crawled_urls.md

```diff
@@ -0,0 +1,23 @@
+# Table: gcp_websecurityscanner_crawled_urls
+
+https://cloud.google.com/security-command-center/docs/reference/web-security-scanner/rest/v1/projects.scanConfigs.scanRuns.findings
```

Comment:
This description link seems off, should it be https://cloud.google.com/security-command-center/docs/reference/web-security-scanner/rest/v1/projects.scanConfigs.scanRuns.crawledUrls/list#CrawledUrl?